src/OpenTelemetry.Api/Context/ThreadLocalRuntimeContextSlot.cs

```diff
@@ -23,9 +24,10 @@ namespace OpenTelemetry.Context
     /// The thread local (TLS) implementation of context slot.
     /// </summary>
     /// <typeparam name="T">The type of the underlying value.</typeparam>
-    public class ThreadLocalRuntimeContextSlot<T> : RuntimeContextSlot<T>
+    public class ThreadLocalRuntimeContextSlot<T> : RuntimeContextSlot<T>, IDisposable

```

Comment:
In this case we probably can put the `IDisposable` in the base class.

Reply:
thanks!